app/webpacker/css/admin/globals/palette.scss

```diff
@@ -0,0 +1,16 @@
+// Basic color palette for admin
+$color-1:                        #FFFFFF !default;    // White
+$color-2:                        #9FC820 !default;    // Green
+$color-3:                        #5498DA !default;    // Light Blue
+$color-4:                        #6788A2 !default;    // Dark Blue
+$color-5:                        #C60F13 !default;    // Red
+$color-6:                        #FF9300 !default;    // Yellow
```

Comment:
I don't really like the naming here. I would prefer something like `background-color`, `foreground-color` and `highlight-color`. But I'll take any weight out of my opinion here because I'm not a designer and that association can happen later, too.

Reply:
And it's not new.

Reply:
Yep, exactly: that's not new. 

I don't know what I prefer actually. I used to change my opinion each 6 months. And I don't think it's that important actually.